hack/terraform/aws/delete_dependency.sh

```diff
@@ -0,0 +1,29 @@
+#! /bin/bash
+# Copyright 2022 Antrea Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+cat $HOME/tmp/terraform-aws/terraform.tfstate | jq > tmp.json
+vpc_id="$( jq '.outputs.vpc_id.value' tmp.json | sed 's/\"//g' )"
```

Comment:
We have vpc id and region as terraform output. Are you doing this because of terraform lock file? Even then you should be able to get those information as `module.vpc.vpc_id` and `var.region` in terraform and pass those in as bash script parameters.